compiler/damlc/daml-stdlib-src/DA/Internal/Prelude.daml

```diff
@@ -515,19 +515,19 @@ unzip3 : [(a, b, c)] -> ([a], [b], [c])
 unzip3 = foldr (\(a,b,c) ~(as,bs,cs) -> (a::as,b::bs,c::cs))
                   ([],[],[])
 
--- | `traceRaw msg a` prints `msg` and returns `a`, for debugging purposes.
+-- | `traceRaw msg a` prints `msg` and returns `a`, for debugging purposes, depending on your loglevel.
```

Comment:
I’m not sure that’s super helpful. For scenarios, triggers and DAML script you always get these. And for sandbox, this doesn’t tell you what you need to change to get the logs. maybe it would be better to extend the documentation for sandbox that explains how you can enable these logs?

Reply:
My problem was that while I happily found these functions, but needed you guys (DAML team) to figure why it does not print anything: I need to set loglevel, and also the way to figure was not so obvious.
This PR is merely a suggestion, I'll leave it to you how to tackle this, please feel free to ignore this PR.
I agree that some docs on loglevel would be useful.

Reply:
How about the following comment (probably on a new line).
Note that for sandbox, you need to increase the log level in your `logback.xml` file to make the messages visible.

```
<logger name="daml.tracelog" level="DEBUG" />
```

Reply:
Sounds helpful. But why the logback? I use the cmd option `--log-level=trace` which is more straightforward to me. Maybe we could refer the reader to docs describing how to configure loglevel? The docs could mention both ways perhaps.